### ISSUE TITLE
⌨ Preferred department or agencies not announcing

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#15220]
### ⌨ Accessibility and Expected Behaviour

When selecting departments/agencies in the "preferred departments or agencies" sections of the career planning the screenreader does not announce any feedback. 

Expected behaviour: an announcement to indicate the number of options

### WCAG 2.1 Criteria

1.3.1 Info and Relationships

### 📋 Steps to Reproduce

1. Sign in
2. verify employee email
3. navigate to career planning
4. Navigate to "your next role" or "your career objective"
5. search for "Canadian space agency" in the preferred departments or agencies section

### 🕵️ Details

🛡️ Fable user reported: After entering text in the editfield dropdown e.g. 'Global affairs', the list populated with the choice that i needed to select. however, there was no feedback that options had appeared. ideally, an alert to indicate the number of options would be helpful.

### What operating systems are you encountering the problem on?

Apple IOS

### What browsers are you encountering the problem on?

Safari

### What assistive technologies are you encountering the problem on?

VoiceOver

### ✅ Acceptance Criteria

- [ ] an announcement / alert to indicate the number of options

**Comment by esizer** [comment]
Easily reproduceable. I think we need to fix it by providing messages 🤔 

https://github.com/downshift-js/downshift?tab=readme-ov-file#geta11ystatusmessage
